roles/default/kiali-deploy/defaults/main.yml

```diff
@@ -256,6 +256,8 @@ kiali_defaults:
     istio_upgrade_action: false
     ui_defaults:
       graph:
+        enable_cytoscape: true
+        enable_patternfly: false
```

Comment:
I'll do it, although I'll change it to just "enable_impl" because it's already under "graph".  I actually already had it this way, so that's annoying :-)